Extension/src/Debugger/attachToProcess.ts

```diff
@@ -63,7 +62,7 @@ export class RemoteAttachPicker {
             let pipeProgram: string = pipeTransport.pipeProgram;
             let pipeArgs: string[] = pipeTransport.pipeArgs;
 
-            let argList = RemoteAttachPicker.createArgumentList(pipeArgs);
+            let argList: string = RemoteAttachPicker.createArgumentList(pipeArgs);
```

Comment:
Do we really need to add a rule to complain about not being explicit everywhere?  A lot of these typings are obvious based on the code (which is probably why they were left out), and VS Code IntelliSense already lets you know the type when the appropriate typings are found.  I find this rule to be a bit too cumbersome.

Reply:
(BTW, I meant to put this comment on line 87 which makes a little more sense than putting it here - sorry!)

Reply:
Are always using explicit types recommended typescript practice? I assume you could omit the type when the compiler is able to determine it, like with "auto" in C++. I think making the type explicit is good for readability when it's unclear, but often times it doesn't really help.

Reply:
Yes, explicit types is recommended Typescript practice. If the type is a dictionary, feel free to use ```any```. However most of the time, the type should be defined.

On the rare chance that this is too much (e.g. https://github.com/WardenGnaw/vscode-cpptools/blob/5c231a544c283a0ef6fb053e703846eff76e350f/Extension/src/LanguageServer/protocolFilter.ts#L13) then disabling the linter is fine.